contents/guides/users/abusive_content.md

```diff
@@ -0,0 +1,20 @@
+---
+title: What can I do if I encounter abusive content or users?
+category: User guides
+---
+
+## Report someone’s post
+
+If you encounter an abusive post, you can click the alert triangle icon at the top right of the post to report it to your podmin. Enter a reason for reporting this post in the dialog box. Please only report posts that break our [community guidelines][community_guidelines] or your pod’s terms of service, for example posts containing illegal content, or which are abusive or spam.
```

Comment:
At the end, "for example posts containing illegal content, abusive content or spam." is more concise and easier to read – but is it clear enough for new users?

Reply:
Yep, your suggestion is easier to read, and I think it's also clear enough.